meta-oe/recipes-connectivity/libwebsockets/libwebsockets_4.3.2.bb

```diff
@@ -24,21 +24,24 @@ PACKAGECONFIG[libuv] = "-DLWS_WITH_LIBUV=ON,-DLWS_WITH_LIBUV=OFF,libuv"
 PACKAGECONFIG[server] = "-DLWS_WITHOUT_SERVER=OFF,-DLWS_WITHOUT_SERVER=ON,"
 PACKAGECONFIG[ssl] = "-DLWS_WITH_SSL=ON,-DLWS_WITH_SSL=OFF,openssl"
 PACKAGECONFIG[static] = "-DLWS_WITH_STATIC=ON,-DLWS_WITH_STATIC=OFF -DLWS_LINK_TESTAPPS_DYNAMIC=ON,"
+PACKAGECONFIG[systemd] = "-DLWS_WITH_SDEVENT=ON,-DLWS_WITH_SDEVENT=OFF,systemd"
```

Comment:
this `PACKAGECONFIG` needs to be enabled as well when systemd is in `DISTRO_FEATURES`

Reply:
I thought of that but ultimately I left it out because it really is an optional feature and shouldn't be automatically enabled just because a distro enables systemd.

Reply:
OK, thats good, however it will be nearly never used by folks since they would assume its always tied to distro features and secondly, since it has dependency on systemd somehow we need to also ensure that this packageconfig when enabled without systemd gives an error if we do not tie it to distro feature.

Reply:
I've added a fixup to check this condition. Please check if that is acceptable or if you would like the error message changed.

Reply:
Please submit that in a new pull request since original patch is already applied

Reply:
> Please submit that in a new pull request since original patch is already applied

I noticed that after I pushed, sorry 😞.

I updated the PR to contain just the new commit, do you want to just pick it or do you want me to open an new PR, or is this one still ok?